plugins/cookie.rst

```diff
@@ -1,4 +1,19 @@
 Cookie Plugin
 =============
 
-TODO: explain the cookie plugin
+Cookie plugins allow you to sore cookies information in a `CookieJar` and reuses them on consequent requests according
```

Comment:
Although there isn't any Cookie documentation IIRC, we should link the Cookie documentation on this page.